extensions/binding/org.eclipse.smarthome.binding.ntp/src/main/java/org/eclipse/smarthome/binding/ntp/handler/NtpHandler.java

```diff
@@ -56,7 +56,7 @@
     private Logger logger = LoggerFactory.getLogger(NtpHandler.class);
 
     /** timeout for requests to the NTP server */
-    private static final int NTP_TIMEOUT = 10000;
+    private static final int NTP_TIMEOUT = 30000;

```

Comment:
If the server really takes 30 secs to respond, I wonder what value the delivered time still has :-/
I would actually think that everything that takes longer than 10 secs is indeed an issue - maybe the user should look for a different ntp server in such a situation.